library/ntc_config_command.py

```diff
@@ -175,16 +159,29 @@ def main():
     secret = module.params['secret']
     use_keys = module.params['use_keys']
     key_file = module.params['key_file']
-    port = int(module.params['port'])
 
     if module.params['host']:
         host = socket.gethostbyname(module.params['host'])
 
-    if connection == 'ssh' and not module.params['host']:
+    if connection in ['ssh', 'telnet'] and not module.params['host']:
```

Comment:
Is this required for ntc_config_command?  We have this in ntc_show_command because you can have connection=offline - maybe it was copied over by accident when module was required...host should be required.